src/OpenSage.Game/Gui/Wnd/Controls/ComboBox.cs

```diff
@@ -258,7 +258,7 @@ protected override void LayoutOverride()
                 0,
                 ClientSize.Height,
                 ClientSize.Width,
-                itemsHeight);
+                itemsHeight + 2);
```

Comment:
Why was this required?

Reply:
Not entirely sure...I added code to not draw ListBoxItems that are outside of the visible area of the ListBox. This lead to the last item not being drawn without these 2 added pixels...I guess there's some kind of border or padding involved.